functions/src/platform/assistant/handler.js

```diff
@@ -170,5 +170,9 @@ module.exports = (actionsMap) => {
     pipeline.stage(pipeline.IDLE);
   });
 
-  return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app), dashbot.configHandler(app));
+  // dashbot.configHandler(app);
+  // return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app));
+  return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app, function (req, res) {
```

Comment:
`bst.Logless.capture` doesn't allow 3rd argument so this code never call callback 
https://github.com/bespoken/logless-client/blob/master/src/logless.ts#L77

Btw, please use arrow function for this callback.